tests/dip_sip/test_dip_sip.py

```diff
@@ -0,0 +1,168 @@
+import pytest
+import ptf.testutils as testutils
+from ipaddress import ip_address
+import logging
+
+TOPO_LIST = {'t0', 't1', 't1-lag'}
+PORTS_TOPO = {'t1'}
+LAG_TOPO = {'t0', 't1-lag'}
+DEFAULT_HLIM_TTL = 64
+WAIT_EXPECTED_PACKET_TIMEOUT = 5
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function', autouse=True)
+def prepare_ptf(testbed_devices):
+    ptfhost = testbed_devices["ptf"]
+    # remove existing IPs from ptf host
+    ptfhost.script('scripts/remove_ip.sh')
+    # set unique MACs to ptf interfaces
+    ptfhost.script('scripts/change_mac.sh')
+
+
+def lag_facts(dut):
```

Comment:
suggest to move to common? @wangxin

Reply:
According to the implementation of this function, I think it is too specific to be common. This function choose two portchannels for testing:
* mg_facts['minigraph_portchannel_interfaces'][2]['attachto']
* mg_facts['minigraph_portchannel_interfaces'][0]['attachto']
Then it collects some information of these chosen port channels from host facts and LLDP facts. It looks more like test specific behavior.